bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
+  File.join 'exercises', slug, '.meta', 'generator'
+}
+
+no_genny_path = ->(slug) { !Dir.exist? genny_path[slug] }
+
+puts Dir.children('exercises')
+  .filter(&no_genny_path).sort.join("\n")
```

Comment:
since `puts` with an array will already print each element on a separate line, the `join("\n")` may be elided if that is desired.

Reply:
Good to know.